src/pages/tutorial/angular/step-1.mdx

```diff
@@ -0,0 +1,919 @@
+---
+title: 1. Installing Carbon
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### Starting with the Carbon Angular, there are two ways to begin working with Carbon components. By the end, these two methods will produce the same result.
+
+<AnchorLinks>
+
+<AnchorLink>Prerequisites</AnchorLink>
+<AnchorLink>Use starter app - recommended</AnchorLink>
+<AnchorLink>Start from scratch</AnchorLink>
+
+</AnchorLinks>
+
+## Preview
+
+A [preview](https://angular-step-2-carbon-tutorial.netlify.com) of what you will build:
+
+<Preview
+  height="200"
+  title="Carbon Angular Tutorial Step 1"
+  src="https://angular-step-2-carbon-tutorial.netlify.com"
```

Comment:
![image](https://user-images.githubusercontent.com/1691245/67425482-aea17c00-f59d-11e9-861e-51916e6d9bec.png)

When on this route, clicking "IBM Carbon Tutorial Angular" reloads the entire page. Not what you'd expect with a SPA.

Reply:
fixed